### PR TITLE
Maryland migration

### PR DESCRIPTION
**Pull request by AudreyAltman**
This adds a mapping for Maryland, migrated from ingestion1.  The mapping has been tested thoroughly with automated reports.  A property value report for `preview` is added to assist with automated testing. The property value report for `isShownAt` is fixed.

One significant change from ingestion1 is that there is no collection mapping.  This is because there doesn't appear to be collection data Maryland's records anymore.  None of the harvested records contained any collection data.

### CODE REVIEW

File: src/main/scala/dpla/ingestion3/mappers/providers/MarylandMapping.scala [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      .flatMap(_.splitAtDelimiter(";"))
+
+  // OreAggregation
+  override def dplaUri(data: Document[NodeSeq]): ZeroToOne[URI] = mintDplaItemUri(data)
+
+  override def dataProvider(data: Document[NodeSeq]): ZeroToMany[EdmAgent] =
+    extractStrings(data \ "metadata" \\ "source")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyAgent)
+      .slice(0, 1) // get first instance
```
Comment (mdellabitta) [inline src/main/scala/dpla/ingestion3/mappers/providers/MarylandMapping.scala]: Would `head` or `headOption` make more sense?

Comment (mdellabitta) [reply src/main/scala/dpla/ingestion3/mappers/providers/MarylandMapping.scala]: This probably doesn't matter at all, just thinking out loud.

Comment (AudreyAltman) [reply src/main/scala/dpla/ingestion3/mappers/providers/MarylandMapping.scala]: Yeah, I just did it that way b/c `slice` returns an empty array if the value is missing, so you don't have to do another step to get the correct return type.

Comment (mdellabitta) [reply src/main/scala/dpla/ingestion3/mappers/providers/MarylandMapping.scala]: TIL. That's awesome!

**Pull request merged**